vim_onenote.ahk

```diff
@@ -117,77 +216,77 @@ Send, {End}{Enter}
 Gosub InsertMode
 return
 
+r::
+    send +{right}
+    gosub, InsertMode
+    ; Wait for single key to be pressed, sends that key and returns to normal
+    input, inp, V E L1
+    send {left}
+    gosub, NormalMode
+return
+
 ; undo
 u::Send, ^z
-return 
 ; redo.
 ^r::Send, ^y
-return 
 
-+G:: Send, ^{End}
 ; G goto to end of document
-return
+sG(){
+    Send, ^{End}
+    }
++G::sG()
 
-g::
+g(){
 ; TBD - Design a more generic way to implement the <command> <motion> pattern. For now hardcode dw. 
 if IsLastKey("g")
-{
     ;gg - Go to start of document
     Send, ^{Home}
-    return
 }
-return
+g::g()
 
-w::
-; TBD - Design a more generic way to implement the <command> <motion> pattern. For now hardcode dw. 
-if IsLastKey("d")
-{
-    ;dw
-    Send, {ShiftDown}
-    ^{Right}
-    Send, {Del}
-    Send, {Shift}
-    return
-}
-if IsLastKey("c")
-{
-    ;cw
-    Send, {ShiftDown}
-    ^{Right}
-    Send, {Del}
-    Send, {Shift}
-    Gosub InsertMode
-    return
-}
-; just w
-Send, ^{Right}
-return 
+s4(){
+    Send, {End} ;$
+    }
++4::s4()
+
+z(){
+    Send, {Home} 
+    }
+0::z()
+
+s6(){
+    Send, {Home} ;^
+    }
++6::s6()
+
+s5(){
+    Send, ^b ;%
+    }
++5::s5()
+
+cF(){
+    Send, {PgDn}
+    }
+^F::cF()
+
+sB(){
+    Send, {PgUp}
+    }
++B::sB()
 
-b::Send, ^{Left}
-return 
-+4::Send, {End}
-return 
-0::Send, {Home}
-return 
-+6::Send, {Home}
-return 
-+5::Send, ^b
-return 
-^f::Send, {PgDn}
-return 
-^b::Send, {PgUp}
 
 
-;; TBD Design a more generic <command> <motion> pattern, for now implement yy and dd the most commonly used commands.
 y::
-if IsLastKey("y")
-{
-    Send, {Home}{ShiftDown}{End}
-    Send, ^c
-    Send, {Shift}
-}
+    SelectMotion()
```

Comment:
probably, because it's a function call. If it were c, the hit would be about 6 clock cycles. IDK about autohotkey. But anyway, it doesn't seem significant, and probably won't be without massive loops.
And thanks, I thought so. Spent a lot of time away from the computer puzzling it out. Tried a couple of other things first.